apps/trading/components/rewards-container/streaks/activity-streaks.tsx

```diff
@@ -0,0 +1,218 @@
+import { VegaIcon, VegaIconNames } from '@vegaprotocol/ui-toolkit';
+import { useT } from '../../../lib/use-t';
+import classNames from 'classnames';
+import { formatNumber } from '@vegaprotocol/utils';
+import BigNumber from 'bignumber.js';
+
+export const ActivityStreak = ({
+  tiers,
+  streak,
+}: {
+  tiers:
+    | never[]
+    | {
+        minimum_activity_streak: number;
+        reward_multiplier: string;
+        vesting_multiplier: string;
+      }[];
+  streak:
```

Comment:
Is it possible to use a generated type here?

Reply:
yes for streaks, for tiers I don't have a generated type as that json is pulled from the network params